extensions/tensorflow/TFExtractTopLabels.cpp

```diff
@@ -156,7 +162,7 @@ int64_t TFExtractTopLabels::TensorReadCallback::process(const std::shared_ptr<io
   auto num_read = stream->read(reinterpret_cast<uint8_t *>(&tensor_proto_buf[0]),
                                    static_cast<int>(stream->size()));
 
-  if (num_read != stream->size()) {
+  if (static_cast<uint64_t>(num_read) != stream->size()) {
```

Comment:
same here: `stream->size()` returns `size_t`, so we should preferably cast `num_read` to that.

Reply:
Updated in 7f2abdc0ae7a9f55c07339edc9c9d84de409c4de